components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/IdentityApplicationManagementClientException.java

```diff
@@ -90,6 +90,31 @@ public IdentityApplicationManagementClientException(String errorCode, String mes
         super(errorCode, message, cause);
     }
 
+    /**
+     * Constructs a new exception with the specified error code, message and description.
+     * @param errorCode Error code.
+     * @param message Error message.
```

Comment:
Param descriptions should be vertically aligned

Reply:
Fixed with e4bf79511c2924cb02a2721d4e99f48a0761156f